src/Raven.Server/Monitoring/Snmp/SnmpOids.cs

```diff
@@ -515,6 +515,9 @@ private Databases()
             [Description("Number of bytes written (documents, attachments, counters, timeseries)")]
             public const string DataWrittenPerSecond = "5.2.{0}.6.2";
 
+            [Description("Number of missing attachments")]
+            public const string CountOfMissingAttachments = "5.2.{0}.6.3";
```

Comment:
The path is `Monitoring/Snmp/Objects/Database/5.2.X/1/DatabaseCountOfMissingAttachments.cs`

so snmp path should be, no?
```suggestion
            public const string CountOfMissingAttachments = "5.2.{0}.1.7";
```

Reply:
Also move it to appropriate line

Reply:
done